js/components/slider.js

```diff
@@ -0,0 +1,61 @@
+var React = require('react'),
+ReactDOM = require('react-dom'),
+Utils = require('./utils');
+
+var Slider = React.createClass({
+
+  componentDidMount: function() {
+    this.handleSliderColoring();
+  },
+
+  componentWillReceiveProps: function(nextProps) {
+    if (nextProps.value != this.props.value) {
+      this.handleSliderColoring();
+    }
+  },
+
+  handleSliderColoring: function(){
+    if (!Utils.isEdge()){
+      var inputs = document.querySelectorAll('input[type=range]');
```

Comment:
Well we were thinking about using sliders for other screens as well, for example CC page. I can imagine several sliders there. Or one slider there and one in control bar.